### PR TITLE
Add mi manual task sample

### PR DESCRIPTION
**Pull request by ThusharaSampath**
https://github.com/wso2-enterprise/choreo/issues/25915

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: .samples/weather-to-logs-mi-manual-task copy.yaml
Comment (chameerar) [inline .samples/weather-to-logs-mi-manual-task copy.yaml:1]: Shall we rename this with a proper name?

File: .samples/weather-to-logs-mi-manual-task copy.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+displayName: Weather Report to Logs - Scheduled Task
+description: Log tabulated weather report with OpenWeatherMap data
+componentType: scheduled-task
+buildPack: wso2-mi
+repositoryUrl: https://github.com/wso2/choreo-samples/
+componentPath: /weather-to-logs-mi-scheduled-task
+thumbnailPath: /icons/weather-to-email-integration.svg
```
Comment (chameerar) [inline .samples/weather-to-logs-mi-manual-task copy.yaml:7]: Should we wait for a proper icon or can we go ahed with this?

**Review by chameerar — CHANGES_REQUESTED** [review]

File: .samples/weather-to-logs-mi-manual-task copy.yaml
Comment (ThusharaSampath) [reply .samples/weather-to-logs-mi-manual-task copy.yaml:1]: fixed with https://github.com/wso2/choreo-samples/pull/88/commits/1c37b76d226164b748cbc7d8ac76735be976a28d

**Review by ruks — APPROVED** [review]

File: .samples/weather-to-logs-mi-manual-task copy.yaml (hunk 1 above)
Comment (ThusharaSampath) [reply .samples/weather-to-logs-mi-manual-task copy.yaml]: we can use this for now 
image

WDYT @chameerar ?

Comment (chameerar) [reply .samples/weather-to-logs-mi-manual-task copy.yaml]: Should be fine for now.

File: .samples/weather-to-email-integration.yaml [hunk 2]
```diff
@@ -8,4 +8,3 @@ thumbnailPath: /icons/weather-to-email-integration.svg
 documentationPath: /weather-to-email-integration/readme.md
 tags:
   - Integration
-
```
Comment (chameerar) [inline .samples/weather-to-email-integration.yaml:11]: Any reason for removing this newline?

Comment (ThusharaSampath) [reply .samples/weather-to-email-integration.yaml:11]: do we need that?

Comment (chameerar) [reply .samples/weather-to-email-integration.yaml:11]: No. Should be fine.

**Review by chameerar — APPROVED** [review]
LGTM

**Pull request merged**